process/charter.html

```diff
@@ -178,13 +178,16 @@ <h2>3. <a id="creation" name="creation">Charter Creation</a></h2>
 
 <h3 id="horizontal-review">3.1 Horizontal Review</h3>
 
-	<p>Once there is agreement on a draft charter, ideally among Team Contact(s), candidate chairs(s), relevant interest groups, and interested community members, seek review from the Strategy Team and horizontal reviewers by:
-	<ol><li>Opening a <a href="https://github.com/w3c/strategy/issues/new?assignees=&labels=Evaluation%3A+untriaged&template=04-Chartering.md&title=">Chartering issue</a> in the <a href="https://github.com/w3c/strategy/issues">Strategy Repo</a>,</li>
+	<p>Once there is agreement on a draft charter, ideally among Team Contact(s), candidate chairs(s), relevant interest groups, and interested community members, seek review from the Strategy Team and horizontal reviewers by:</p>
+	<ol>
+  <li>Moving/Copying the draft charter into the <a href="https://github.com/w3c/charter-drafts/">charter-drafts Repo</a>, naming it using the convention [YYYY]/[shortname-name]-[group-type].html .</li>
+  <li>Opening a <a href="https://github.com/w3c/strategy/issues/new?assignees=&labels=Evaluation%3A+untriaged&template=04-Chartering.md&title=">Chartering issue</a> in the <a href="https://github.com/w3c/strategy/issues">Strategy Repo</a>,</li>
 	<li>Adding a card for the issue to the <a href="https://github.com/w3c/strategy/projects/2">Incubation Pipeline</a>,</li>
 	<li>Adding the <a class="IssueLabel-big" style="background: #9ae504;" href="https://github.com/w3c/strategy/issues">Horizontal review requested</a>
 	label to the issue, and</li>
-	<li>Optionally, announce the new charter-in-progress to team-strategy and/or on a Strategy Team call.</li>
-	</ol></p>
+  <li>Creating an issue in <a href="https://github.com/w3c/AB-memberonly">w3c/AB-memberonly</a> to alert the AB and our Members.</li>
+  <li>Optionally, announce the new charter-in-progress to team-strategy and/or on a Strategy Team call.</li>
```

Comment:
```suggestion
  <li>Optionally, announcing the new charter-in-progress to team-strategy and/or on a Strategy Team call.</li>
```

Reply:
Also, is "team-strategy" something that can be linked, e.g., is it a mailing list?